react/src/components/EndpointSelect.tsx

```diff
@@ -50,21 +52,31 @@ const EndpointSelect: React.FC<EndpointSelectProps> = ({
             ...EndpointLLMChatCard_endpoint
           }
         }
+        endpoint(endpoint_id: $endpoint_id) @skip(if: $skipEndpoint) {
```

Comment:
```suggestion
        endpoint(endpoint_id: $endpoint_id) @skipOnClient(if: $skipEndpoint) {
```

Reply:
In this case, it's better to use `@skip`. `@skipOnClient` is a workaround to maintain compatibility for multiple schema versions.